tests/Commands/MakeEnumTest.php

```diff
@@ -175,12 +175,13 @@ public function it_can_make_an_enum_with_values_const(): void
 
     private function runMakeCommand(string $stub, array $arguments = []): void
     {
+        $WeekDayEnum = "WeekDay";
```

Comment:
Variables should be lowerCameCase and in tests it's also not necessary to prevent duplicated code or strings in every case.